workers/unity/Assets/Fps/Scripts/GameLogic/WorldTiles/TileFootprintHelper.cs

```diff
@@ -0,0 +1,89 @@
+﻿using UnityEngine;
+#if UNITY_EDITOR
+using UnityEditor;
+using UnityEditor.Experimental.SceneManagement;
+
+#endif
+
+
+public class TileFootprintHelper : MonoBehaviour
+{
+#if UNITY_EDITOR
+    public float TileSize = 8 * 4;
```

Comment:
have the tile sizes changed now? I thought they were 36x36

Reply:
the full footprint of the tile is 36m (8 * 4m 'squares' + a 4m gutter square), this helper is the inner part of the tile, the 8 * 4m part.

Reply:
In the new branch, I've renamed the var to `InnerTilerSize` to make it clearer what it represents